pkcs11/pkcs11.go

```diff
@@ -1721,13 +1721,14 @@ func (r *rsaPrivateKey) getHash() *crypto.Hash {
 		return r.hash
 	}
 	// Initialized to SHA256 if no hash specified
```

Comment:
You should delete this comment, it is already out of sync with the code ;)

Reply:
Ahh, I meant to change line 1724 back to sha256. I've just updated the code to make the comment relevant again.